packages/suite/src/views/wallet/coinmarket/common/CoinmarketTransactionId.tsx

```diff
@@ -19,6 +18,8 @@ const Wrapper = styled.div`
 
 const Left = styled.div`
     display: flex;
+    padding-right: 10px;
```

Comment:
Btw in general we try to use spacing tokens for all these margin/padding/positioning stuff: `spacings.xs`

Reply:
done